tests/parser/FieldPropsTests.cpp

```diff
@@ -134,6 +134,58 @@ PERMX
 }
 
 
+
+BOOST_AUTO_TEST_CASE(CreateFieldPropsForActnum) {
+    std::string deck_string = R"(
+GRID
+
+BOX
+  1 3 1 3 1 1 /
+
+ACTNUM
+   9*0 /
+
+BOX
+  1 3 1 2 1 1 /
+
+ACTNUM
+   6*1 /
+
+ENDBOX
+
+-- Re-enable (3,3,1)
+EQUALS
+    ACTNUM 1 3 3 3 3 1 1 /
+/
+)";
+
+    Deck deck = Parser{}.parseString(deck_string);
+    EclipseGrid grid{GridDims{3, 3, 1}};
+    FieldProps fp(deck, grid);
+    std::vector<int> expected_actnum = { 1, 1, 1,
+                                         1, 1, 1,
+                                         0, 0, 1  };
+    auto actnum = fp.actnumRaw();
+    BOOST_TEST(actnum == expected_actnum);
```

Comment:
Any reason not to write this in terms of `BOOST_CHECK_EQUAL_COLLECTIONS`?  That way you get additional information regarding positions if elements differ
```C++
BOOST_CHECK_EQUAL_COLLECTIONS(actnum.begin(), actnum.end(), expected_actnum.begin(), expected_actnum.end());
```

Reply:
I will change it.

Reply:
Done